src/layers/core/arc-layer/arc-layer.js

```diff
@@ -18,11 +18,12 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {Layer} from '../../../lib';
-import {assembleShaders} from '../../../shader-utils';
 import {GL, Model, Geometry} from 'luma.gl';
+import {join} from 'path';
+import {readFileSync} from 'fs';
 
-const glslify = require('glslify');
+import {Layer} from '../../../lib';
```

Comment:
These are imported differently in other layers (using single import line), let's be consistent.